src/components/remote_control/src/commands/base_command_request.cc

```diff
@@ -473,6 +366,48 @@ bool BaseCommandRequest::CheckDriverConsent() {
   return false;
 }
 
+bool BaseCommandRequest::AqcuireResources() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  const Json::Value message_params =
+      MessageHelper::StringToValue(message_->json_message());
+
+  if (!IsResourceFree(ModuleType(message_params))) {
+    LOG4CXX_WARN(logger_, "Resource is busy.");
+    SendResponse(false, result_codes::kInUse, "");
+    return false;
+  }
+
+  AcquireResult::eType acquire_result = AcquireResource(message_params);
+  switch (acquire_result) {
+    case AcquireResult::ALLOWED: {
+      SetResourceState(msg_json_, ResourceState::BUSY);
+      return true;
+    }
+    case AcquireResult::IN_USE: {
+      SendResponse(false, result_codes::kInUse, "");
+      return false;
+    }
+    case AcquireResult::ASK_DRIVER: {
+      SetResourceState(msg_json_, ResourceState::BUSY);
+
+      Json::Value value;
+      Json::Reader reader;
+      LOG4CXX_DEBUG(logger_, "Request: " << message_->json_message());
+      reader.parse(message_->json_message(), value);
+
+      SendGetUserConsent(value);
+
+      return false;
```

Comment:
@dev-gh again double response. In that case you will send concent and response with  SendResponse(false, result_codes::kInvalidData, "");

In is not correct